app/views/unattended/provisioning_templates/snippet/redhat_register.erb

```diff
@@ -173,6 +173,8 @@ description: |
       <%= "subscription-manager repo-override --list --repo $repo | grep 'cost:' &>/dev/null || subscription-manager repo-override --repo $repo --add=cost:#{host_param('subscription_manager_override_repos_cost')}" %>
     done
   <% end -%>
+  subscription-manager refresh
+  subscription-manager status
```

Comment:
@parthaa Do we want to put these commands under a parameter switch? My concern is that we are increasing the run time for this snippet for all scenarios.

Also: could it cause other issues?

Reply:
Personally, I find the extra logging and status information invaluable when troubleshooting subscription issues later in the install process.  My local running of this increases overall time by about 1 second.

Reply:
+1 to putting it behind a host parameter. And I'm not sure why the refresh would be needed? You've just registered, so your certificates are almost guaranteed to be up-to-date.

Reply:
Weirdly, my RHEL8 boxes seem to require this refresh.  I've no objection to putting it behind some sort of flag.  Any suggested names?